test/util.py

```diff
@@ -402,6 +402,20 @@ def validate_cwl(cwl: str, cmd: Command = None, explore: bool = True):
             assert len(parsed["outputs"]) == len(cmd.outputs) + 1
 
 
+def validate_janis(janis: str, cmd: Command = None):
+    fn = "temp-janis"
+    if cmd and cmd.as_filename:
+        fn = cmd.as_filename
+    with tempfile.TemporaryDirectory() as tmpdir:
+        tmpdir = Path(tmpdir)
+        tmpfile = tmpdir / (fn + ".py")
+        tmpfile.write_text(janis)
+
+        # just run the python file, and if it succeeds we're sweet
+        result = subprocess.run(["python", tmpfile])
```

Comment:
I assume this is running the class definition. It's checking syntax which is good. Does it do any other validation?

Reply:
When converting to Janis, a translate is inserted at the bottom of the file, so running the python file will actually also translate it to WDL.

```python
if __name__ == "__main__":
    # or "cwl"
    YourTool().translate("wdl")
```

Reply:
Sounds good!